packages/datadog-plugin-cucumber/test/index.spec.js

```diff
@@ -180,6 +195,10 @@ describe('Plugin', () => {
                 const parentCucumberStep = testTrace.find(span => span.meta['cucumber.step'] === 'integration')
                 expect(httpSpan.parent_id.toString()).to.equal(parentCucumberStep.span_id.toString())
               }
+              if (testName === 'hooks fail') {
```

Comment:
These conditions are starting to pile up in test framework tests. Any way to split that in multiple tests instead?

Reply:
yeah true. The setup made sense when all the tests were checking the same, but not anymore. I'll change it

Reply:
done for cucumber: https://github.com/DataDog/dd-trace-js/pull/1429/commits/0133537ab804cfb94418c222652f6759b40837ba
done for mocha: https://github.com/DataDog/dd-trace-js/pull/1429/commits/596fef9f7c8769e688624f55c4970bcdabc337b9

jest already had multiple tests